docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -187,7 +187,7 @@ const LegacyChip = styled(function LegacyChip(props) {
     },
     '& .MuiChip-label': { px: 0.6 },
     fontSize: (theme) => theme.typography.pxToRem(10),
-    fontWeight: 600,
+    fontWeight: 'semiBold',
```

Comment:
So we use the design token, not raw values. This is `theme.typography.fontWeightSemiBold`.

Reply:
I'm not a huge fan of this API though. Sometimes, it feels like we are using this API https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#values. A possible solution would be to have distinguishing API like $ https://stitches.dev/docs/tokens#using-tokens

```jsx
fontWeight: '$semiBold',
```

But this is off-topic to this PR.